commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/TwoSidedDrive.scala

```diff
@@ -36,7 +36,8 @@ trait TwoSidedDriveProperties extends UnicycleProperties {
 /**
   * A drivetrain with two side control (such as a tank drive)
   */
-trait TwoSidedDrive extends UnicycleDrive { self =>
+abstract class TwoSidedDrive(updatePeriod: Time)(implicit clock: Clock)
```

Comment:
Wouldn't we just want the update period to just be the robot's update period? Why would we want to be able to configure it?